src/pages/Registration/Steps/Documentation/Form/useSubmit.ts

```diff
@@ -57,5 +60,5 @@ export default function useSubmit() {
       handleError
     );
   };
-  return { submit: handleSubmit(submit), isSubmitting };
+  return { submit: handleSubmit(submit), getValues, isSubmitting };
```

Comment:
note that `getValues` only get `hqCountry` once, if in `Documentation`, user changes `hqCountry` - `getValues("hqCountry")` would still be the same,